resources/homepage.html

```diff
@@ -78,6 +108,25 @@ <h1>MFAH Online Portal</h1>
         <a href="/member/register">Member Registration</a>
         <a href="/employee/register">Employee Registration</a>
         <a href="/logout">Logout</a>
+        <!-- Create the dropdown menu -->
```

Comment:
Drop down menus need to be located in the appropriate home pages for either the members or the employees depending on who needs to access the material. Currently they are accessible to everyone who accesses the website without even logging in.

Additionally, the reports need to allow the users to specify different constraints to modify the report that is produced. Ie, "start date to end date" "revenue for museum x only" etc.